README.md

```diff
@@ -65,6 +67,7 @@ jobs:
         with:
           cache-source: var-lib-apt
           cache-target: /var/lib/apt
+          skip-extraction: ${{ steps.cache-var-lib-apt.outputs.cache-hit }}
```

Comment:
This has to be a separate PR, as `skip-extraction` is missing in `buildkit-cache-dance@v2.1.3`

Reply:
Done ✅